sdv/metadata/multi_table.py

```diff
@@ -138,7 +138,7 @@ def _validate_circular_relationships(
 
         for child in children:
             if child in parents:
```

Comment:
minor: can we rename parents to visited? I was confused reading this thinking that we were checking the parents of only the current node, when this list actually keeps growing.